tasks/kernel_matrix_testing/env-setup.sh

```diff
@@ -4,34 +4,45 @@ set -eo xtrace
 
 DIR_NAME=$(dirname "$0")
 
-# Install dependencies
-
-sudo apt update
-sudo apt install -y \
-    aria2 \
-    fio \
-    socat \
-    qemu-kvm \
-    libvirt-daemon-system \
-    curl \
-    debootstrap \
-    libguestfs-tools \
-    libvirt-dev \
-    python3-pip \
-    nfs-kernel-server \
-    rpcbind \
-    ssh-askpass \
-    xsltproc
-
-if [ "$(uname -m )" == "aarch64" ]; then
-    sudo apt install -y qemu-efi-aarch64
+os_family=$(uname)
+
+if [ "$os_family" == "Linux" ]; then
```

Comment:
Maybe we can have separate scripts for linux and macos and call the appropriate one here.